### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by Omkaragrawal**
Updated Readme. Added a few details, specially regarding DB_SSL and MAIL_SSL.

### CODE REVIEW

**Comment by vantezzen** [comment]
Thank you for your contribution!

File: README.md [hunk 1]
```diff
@@ -57,8 +57,10 @@ After setting up the docker container we recommend you to set up an NGINX revers
 	    -e DB_USER=<yourdbuser> \
 	    -e DB_PASSWORD=<yourdbpass> \
 	    -e DB_DATABASE=<yourdbdatabase> \
+	    -e DB_SSL=false \
```
Comment (Omkaragrawal) [inline README.md:60]: Hey @vantezzen  I am don't know much about docker, so please look at this, if it is fine.

File: README.md [hunk 2]
```diff
@@ -57,8 +57,10 @@ After setting up the docker container we recommend you to set up an NGINX revers
...
 	    -e IS_CREATION_ENABLED=true \
 	    -e CONNECT_WITH_FRANZ=true \
+      -e MAIL_SSL=true/false \
```
Comment (Omkaragrawal) [inline README.md:63]: Hey @vantezzen  I am don't know much about docker, so please look at this, if it is fine.

File: README.md [hunk 3]
```diff
@@ -84,10 +86,12 @@ After setting up the docker container we recommend you to set up an NGINX revers
         - DB_USER=<yourdbuser>
         - DB_PASSWORD=<yourdbpass>
         - DB_DATABASE=<yourdbdatabase>
+        - DB_SSL=true/false
```
Comment (Omkaragrawal) [inline README.md:89]: Hey @vantezzen  I am don't know much about docker, so please look at this, if it is fine.

File: README.md [hunk 4]
```diff
@@ -84,10 +86,12 @@ After setting up the docker container we recommend you to set up an NGINX revers
...
         - IS_CREATION_ENABLED=true/false
         - CONNECT_WITH_FRANZ=true/false  
         - IS_REGISTRATION_ENABLED=true/false  
-        - IS_DASHBOARD_ENABLED=true/false  
+        - IS_DASHBOARD_ENABLED=true/false
+        - MAIL_SSL=true/false
```
Comment (Omkaragrawal) [inline README.md:94]: Hey @vantezzen  I am don't know much about docker, so please look at this, if it is fine.

**Pull request merged**